src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,51 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String input) {
+        if (nullOrEmpty(input)) {
+         return 0;
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다. IDE로 IntelliJ IDEA를 사용 중이시라면, 
단축키(Windows는`Ctrl + Alt + L`, Mac은 `Cmd + Option + L`)를 활용해보시는 걸 권장 드립니다.
```suggestion
            return 0;
```